core/whistle_config/src/whapps_config.erl

```diff
@@ -223,6 +260,12 @@ get_ne_binary(Category, Key, Default, Node) ->
 -spec get(config_category(), config_key(), Default) -> any() | Default.
 -spec get(config_category(), config_key(), Default, ne_binary() | atom()) -> any() | Default.
 
+-ifdef(TEST).
```

Comment:
Shouldn't this be the only macro test? All the other accessors (`get_*`) call get/2 or get/4, no?